addons/l10n_ar/models/account_move_line.py

```diff
@@ -10,14 +10,15 @@ def _l10n_ar_prices_and_taxes(self):
         self.ensure_one()
         invoice = self.move_id
         included_taxes = self.tax_ids.filtered('tax_group_id.l10n_ar_vat_afip_code') if self.move_id._l10n_ar_include_vat() else False
+        price_digits = 10**self.env['decimal.precision'].precision_get('Product Price')
         if not included_taxes:
             price_unit = self.tax_ids.with_context(round=False).compute_all(
-                self.price_unit, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)
-            price_unit = price_unit['total_excluded']
+                self.price_unit * price_digits, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)
+            price_unit = price_unit['total_excluded'] / price_digits
```

Comment:
Your problem here is `total_excluded` is already rounded whatever the `round` parameter in the context. Adding `raw_total_excluded` in `compute_all` is a better fix because you are creating rounding issues here.

@jco-odoo What's the point of this method? Is it all about excluding some taxes from the invoice? On later version, this won't work with the round globally taxes computation method.

Reply:
@smetl @jco-odoo I know that it is ugly but I did the fix with a similar approach to that done in 13 in l10n_latam_invoice_document . https://github.com/odoo/odoo/blob/13.0/addons/l10n_latam_invoice_document/models/account_move_line.py#L36:L37 . The goal of this pr is to show on the printed report the same quantity of digits set on "Product Price" Decimal Accuracy when Argentinean invoice report is printed and not to create rounding issues here. Please tell me if you suggest to make the fix with other way.

Reply:
@smetl We need this side because the price can be tax included and you need without and opposite.  We do not have a round False option anymore?  (or maybe in 18?)